stubs/RdKafka/KafkaConsumer.php

```diff
@@ -79,6 +79,16 @@ public function getSubscription()
     {
     }
 
+    /**
+     * @param string    $topic_name
+     * @param TopicConf $topic_conf
+     *
+     * @return Topic
```

Comment:
I'm wondering if we should be more specific here. The docs say:

"Returns a [RdKafka\ConsumerTopic](https://arnaud.le-blanc.net/php-rdkafka-doc/phpdoc/class.rdkafka-consumertopic.html)."

Reply:
@kwn yep, thats my mistake, it should by KafkaConsumerTopic according to this https://github.com/arnaud-lb/php-rdkafka/blob/6.x/kafka_consumer.stub.php#L52